karma.conf.js

```diff
@@ -6,37 +7,72 @@ module.exports = (config) => {
     customLaunchers: {
       SlimChromeHeadless: {
         base: 'ChromeHeadless',
-        flags: ['--headless', '--disable-gpu', '--disable-translate', '--disable-extensions']
+        flags: [
+          '--headless',
+          '--disable-gpu',
+          '--disable-translate',
+          '--disable-extensions'
+        ]
       }
     },
     files: [
-      { pattern: 'src/*.spec.js', watched: false }
+      {
+        pattern: 'src/*.spec.js',
```

Comment:
wouldn't this be duplicate of the one 👇 I thought that globbing with `**/*`means any level from 0 to any.